composer.json

```diff
@@ -13,14 +13,18 @@
         {
             "name": "Jérémy Marodon",
             "email": "marodon.jeremy@gmail.com"
+        },
+        {
+            "name": "Peter Lee",
+            "email": "peter279k@gmail.com"
         }
     ],
     "require": {
         "php": "^5.6 || ^7.0",
-        "fabpot/goutte": "^3.1"
+        "guzzleHttp/guzzle": "^6.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.4.3",
+        "phpunit/phpunit": "^5.5",
```

Comment:
Accodring to the [PHPUnit](https://phpunit.de/) official website,the version 5.5 is now currently released and it supports the PHP 7 and PHP5.6.